src/server/routes/api/index.js

```diff
@@ -139,93 +157,120 @@ function constructPatronObject(object) {
   };
 }
 
-function isTokenExipring(expirationTime, timeThreshold = 5, type = 'minutes') {
-  return (expirationTime.diff(moment(), type) < timeThreshold);
+function isTokenExipring(expirationTime, timeThreshold = 5, type = "minutes") {
+  return expirationTime.diff(moment(), type) < timeThreshold;
 }
 
-export function initializeAppAuth(req, res, next) {
-  req.app.get('logger').info('initializeAppAuth');
-  const tokenObject = req.app.get('tokenObject');
-  const tokenExpTime = req.app.get('tokenExpTime');
+// App-level cache object for API token related variables to be used in
+// `initializeAppAuth` and `createPatron`.
+const app = {};
+
+export async function initializeAppAuth(req, res) {
+  // TODO: Update the logger. `req.app` no longer exists and was used
+  // specifically from express.
+  // req.app.get('logger').info('initializeAppAuth');
+  const tokenObject = app["tokenObject"];
+  const tokenExpTime = app["tokenExpTime"];
```

Comment:
This is the change I actually made (not prettier). `req.app` is no longer a thing so I made the local cache be an object called `app`.